### PR TITLE
ChromeOS count

### PR DESCRIPTION
**Pull request by rudibravo**
Please remember apply the changes on Big Query UI
@Rise-Vision/delivery please review

### CODE REVIEW

**Comment by rudibravo** [comment]
[(2) Update Klipfolio to include new Chrome OS Players](https://trello.com/c/RU9pJs6j/4613-2-update-klipfolio-to-include-new-chrome-os-players)

File: projects/client-side-events/datasets/Installer_Events/views/DisplayCountByInstallerType.bq [hunk 1]
```diff
@@ -542,7 +543,74 @@ FROM (
...
+                  AND os_description LIKE "Android%")
+                OR player_name="Cenique"
+                OR ((player_name = ""
+                    OR player_name IS NULL)
+                  AND player_version LIKE "2.0%")
+                OR player_name="RisePlayerPackagedApp"
+                OR player_name="RisePlayerElectron"
+                OR (player_name="RisePlayer"
+                  AND (player_version LIKE "2015%"
+                    OR player_version LIKE "2016%")))
```
Comment (tejohnso) [inline projects/client-side-events/datasets/Installer_Events/views/DisplayCountByInstallerType.bq]: Is all this exclusion detail about making sure we're not counting old players that were on a different platform? 
Are the 2015 and 2016 dates important? What about 2017?

File: projects/client-side-events/datasets/Installer_Events/views/DisplayCountByInstallerType.bq [hunk 2]
```diff
@@ -26,14 +26,14 @@ FROM (
             REGEXP_EXTRACT(protoPayload.resource,r'\/v2\/viewer\/display\/([A-Za-z0-9-]+)') AS display_id,
```
Comment (tejohnso) [inline projects/client-side-events/datasets/Installer_Events/views/DisplayCountByInstallerType.bq]: I don't understand this "Updated query with bigquery version" commit. It looks like it's including one extra day of history. Nothing about versions.

Comment (rudibravo) [reply projects/client-side-events/datasets/Installer_Events/views/DisplayCountByInstallerType.bq]: I just copied the view from bigquery, it was not matching the version on git

File: projects/client-side-events/datasets/Installer_Events/views/DisplayCountByInstallerType.bq (hunk 1 above)
Comment (rudibravo) [reply projects/client-side-events/datasets/Installer_Events/views/DisplayCountByInstallerType.bq]: Yes, those are exclusions. This is the definition for a v2 player

`player_name="RisePlayer" AND (player_version LIKE "2015%" OR player_version LIKE "2016%"`

I don't think we have RisePlayer v2 on year 2017 onward.

**Pull request merged**